lib/puppet/provider/service/launchd.rb

```diff
@@ -6,8 +6,7 @@
 
     For `launchd` documentation, see:
 
-    * <https://developer.apple.com/macosx/launchd.html>
-    * <http://launchd.macosforge.org/>
+    * <https://developer.apple.com/library/content/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/CreatingLaunchdJobs.html>
```

Comment:
It's hard for me to tell whether this is an equivalent link. Might also include the launchd man page https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man8/launchd.8.html

Reply:
I'll replace.

There is no truly equivalent link because it looks like launchd was removed from Mac OS Forge with OS X 10.10 and moved it to the closed-source libXPC project.